web/modules/custom/hoeringsportal_audit_log/src/Helpers/ConfigHelper.php

```diff
@@ -124,20 +147,26 @@ public function isConfigActive(string $currentRouteName, string $entityTypeId, ?
     $types = $this->getConfiguration('types');
 
     // If no types configuration exists, return an empty array immediately.
-    if (!$types) {
+    if (!$types && is_array($types)) {
```

Comment:
Why do you check if `$types` is an array here? Isn't it being empty enough for an early return?